runtime/src/snapshot_utils.rs

```diff
@@ -197,10 +197,10 @@ pub fn package_snapshot<P: AsRef<Path>, Q: AsRef<Path>>(
 
 fn get_archive_ext(archive_format: ArchiveFormat) -> &'static str {
```

Comment:
Going from extension to ArchiveFormat didn't include the leading `.`, so I removed it here for symmetry.